src/expressions/functions/builtInFunctions_identifiers.ts

```diff
@@ -9,28 +9,23 @@ import { NODE_TYPES } from '../../domFacade/ConcreteNode';
 import FunctionDefinitionType from './FunctionDefinitionType';
 
 function findDescendants(domFacade, node, isMatch) {
-	const results = domFacade
-		.getChildNodePointers(node)
-		.reduce(function (matchingNodes, childNode) {
-			Array.prototype.push.apply(
-				matchingNodes,
-				findDescendants(domFacade, childNode, isMatch)
-			);
-			return matchingNodes;
-		}, []);
+	const results = domFacade.getChildNodePointers(node).reduce((matchingNodes, childNode) => {
+		Array.prototype.push.apply(matchingNodes, findDescendants(domFacade, childNode, isMatch));
```

Comment:
It may be better to push in a loop here - apply with very large arrays risks a stack overflow